packages/omgx/wallet/contracts/LP/L1LiquidityPool.sol

```diff
@@ -4,7 +4,7 @@ pragma solidity >0.5.0;
 pragma experimental ABIEncoderV2;
 
 import "./interfaces/iL2LiquidityPool.sol";
-import "./libraries/OVM_CrossDomainEnabled.sol";
+import "@eth-optimism/contracts/contracts/optimistic-ethereum/libraries/bridge/OVM_CrossDomainEnabled.sol";
```

Comment:
Implement this customized contract
https://github.com/omgnetwork/optimism/blob/develop/packages/omgx/message-relayer-fast/contracts/libraries/OVM_CrossDomainEnabled.sol
 with the L1_LiquidityPool instead, in order to have provision for using the custom messenger.
 
reason - https://github.com/omgnetwork/optimism/tree/develop/packages/omgx/message-relayer-fast#using-the-custom-messenger

Reply:
Yes! Change/fix awaits the `message-relayer-fast` and its OVM_CrossDomainEnabledFast.sol